check_ldap_monitor.pl

```diff
@@ -384,24 +390,28 @@ sub get_value {
     my ( $ldapconn, $base, $scope, $filter, $attribute ) = @_;
     my $message;
     my $entry;
+    my $attrs = ref($attribute) eq 'ARRAY' ? $attribute : [$attribute];
     $message = $ldapconn->search(
         base   => $base,
         scope  => $scope,
         filter => $filter,
-        attrs  => [$attribute]
+        attrs  => $attrs
     );
     $message->code
       && &verbose( '1', $message->error )
       && return ( $message->code, $message->error );
     if ( $entry = $message->shift_entry() ) {
-        my $result = $entry->get_value($attribute);
-        unless ($result) {
-            &verbose( 1, "Attribute $attribute not found in " . $entry->dn );
-            return ( 1, "Attribute $attribute not found" );
+        my @result = ();
+        for (@$attrs) {
+            push(@result, $entry->get_value($_));
         }
-        &verbose( '2', "Found value $result" );
+        unless (@result > 0) {
+            &verbose( 1, "Attribute $attrs not found in " . $entry->dn );
+            return ( 1, "Attribute $attrs not found" );
```

Comment:
Not sure this log message will work if `$attrs` is an array. Maybe we stop in the previous loop when on of the attribute is not found.